apps/openchallenges/challenge-service/src/main/resources/db/migration/V1.0.0__create_tables.sql

```diff
@@ -71,7 +82,7 @@ CREATE TABLE `challenge_incentive`
 CREATE TABLE `challenge_submission_type`
 (
     `id`                    int NOT NULL AUTO_INCREMENT,
-    `name`                  ENUM('container_image', 'prediction_file', 'other'),
+    `name`                  ENUM('container_image', 'prediction_file', 'notebook', 'other'),
```

Comment:
This line should already be in `main`. I'm wondering why GH is still flagging it.